atom/browser/web_contents_preferences.cc

```diff
@@ -230,6 +230,22 @@ bool WebContentsPreferences::UsesNativeWindowOpen(
   return use;
 }
 
+bool WebContentsPreferences::IsPluginsEnabled(
```

Comment:
Its fine to have this helper, but like you suggested it would be better to have them unified into a single `IsPreferenceEnabled`. @kevinsawicki thoughts ?